drivers/gles3/storage/material_storage.cpp

```diff
@@ -2320,11 +2320,14 @@ void MaterialStorage::global_shader_uniforms_instance_update(RID p_instance, int
 		ShaderLanguage::TYPE_VEC3, // vec3
 		ShaderLanguage::TYPE_IVEC3, //vec3i
 		ShaderLanguage::TYPE_MAX, //xform2d not supported here
+		ShaderLanguage::TYPE_VEC4, //vec4
+		ShaderLanguage::TYPE_MAX, //ivec4 not supported here
```

Comment:
I guess `TYPE_IVEC4` could be added using `Vector4i` now that it's available?

Reply:
Ah, yes, its supported. Fixed.